README.md

```diff
@@ -345,3 +345,14 @@ If you want to contribute to this project, see [Contributing Guide](https://gith
+
+**Note** - the default setting for Akka 2.5 is `akka.actor.allow-java-serialization=on`, whereas `akka.actor.allow-java-serialization=off` is the default setting for Akka 2.6
```

Comment:
Actually! I'd move that to **before** the table, it'd be harder to overlook

Reply:
Does it look nice now?